admin/devurls.md

```diff
@@ -22,35 +22,40 @@ your cluster, you'll need to modify your:
 
 ### Step 1: Modify the Helm chart
 
-Set `devurls.host` to a wildcard domain pointing to your ingress controller:
+Set `devurls.host` to a wildcard domain:
 
 ```shell
-helm upgrade coder coder/coder --set devurls.host="*.my-custom-domain.io"
+helm upgrade coder coder/coder --set coderd.devurlsHost="*.my-custom-domain.io"
 ```
 
-If you're using the default ingress controller, specifying a value for
-`devurls.host` automatically adds a rule that routes dev URL requests to the
-user's workspace:
+**Note:** If you are providing an ingress controller, then you will need to add
+the rule manually to the ingress. If you are unsure, skip this step:
 
 ```yaml
  - host: "*.my-custom-domain.io"
     http:
       paths:
       - path: /
         backend:
-          serviceName: envproxy
+          serviceName: coderd
           servicePort: 8080
 ```
 
-If you are providing an ingress controller, then you will need to add the rule
-manually.
-
 ### Step 2: Modify the wildcard DNS record
 
 The final step to enabling dev URLs is to update your wildcard DNS record. Get
-the **ingress IP address** using `kubectl --namespace coder get ingress` and
-point your wildcard DNS record (e.g., \*.my-custom-domain.io) to the ingress IP
-address.
+the **LoadBalancer IP address** using `kubectl --namespace coder get svc` and
+point your wildcard DNS record (e.g., \*.my-custom-domain.io) to the
+LoadBalancer's IP address.
```

Comment:
Do we want to mention the line is `ingress-nginx`? I'm assuming it's the external IP address not the cluster-IP?